packages/ra-ui-materialui/src/detail/Show.spec.tsx

```diff
@@ -126,6 +132,11 @@ describe('<Show />', () => {
         expect(screen.getByTestId('custom-component')).toBeDefined();
     });
 
+    it('should display a custom title', async () => {
+        render(<CustomTitle />);
+        await screen.findByText('book_1_test');
+    });
+
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏Why not add tests for Edit too?